design/process/review-guidelines.md

```diff
@@ -8,40 +8,55 @@ editor: markdown
 dateCreated: 2021-05-28T21:02:49.613Z
 ---
 
-The design team posts new specs to [GitHub Discussions](https://github.com/centerofci/mathesar/discussions/categories/review-request) when there are new designs to review. All [core team](/team) members are encouraged to review new designs. This page describes the process.
+The design team creates specs to describe the design solutions related to the Mathesar product. Whether small or large, all design issues should be accompanied by a spec containing the solution details and information relevant to its implementation. Every spec goes through a review process so that the team can discuss it and ask questions.
 
-# Review Process
+The spec creation process begins when a design issue is started and finalizes when the spec is approved and merged. When a design issue is ready for review, a spec is created, and a PR is opened. Finalizing a design spec means, in most cases, that the design issue has also been solved. If this is not the case, the reason and additional requirements must be communicated in the design issue and the spec.
 
-## Authors
-- Write your design spec on the wiki under the [Documents](/design/specs) folder and link to it from the top-level [Documents](/design/specs) page.
+## Review Process
```

Comment:
Why make this a second level heading? We don't have any first level headings.

I think it would also be good to explain the review process under the heading here. Maybe move this from the first paragraph:

> Every spec goes through a review process so that the team can discuss it and ask questions.

I think it would also be good to explain the goals of the review process here. _Why_ is the team discussing it and asking questions? I can think of some answers: to make sure the design is technically feasible, to ensure that all the scenarios needed for implementation are covered, to look at the design from the perspective of a new or advanced user and imagine using the product, etc. You may be able to think of more reasons. I think having those here will help reviewers.